src/procedures/procedures.11tydata.yaml

```diff
@@ -6,23 +6,24 @@ procedureStandard:
       image: plug-ethernet.jpeg
       content: |
         <ul>
-          <li>Plug in your Ethernet cable and make sure it is locked into place</li>
-          <li>The other end of the Ethernet cable will need to be connected to your Router or Switch</li>
+          <li>Plug in your Ethernet cable and make sure it is locked into place.</li>
+          <li>The other end of the Ethernet cable will need to be connected to your Router or Switch.</li>
         </ul>
     - title: Power it up!
       image: plug-ethernet-power.jpeg
       content: |
         <ul>
-          <li>Connect the power to the wall and then to the Home&nbsp;Assistant Yellow</li>
+          <li>Connect the power to the wall and then to the Home&nbsp;Assistant Yellow.</li>
         </ul>
     - title: Get started with Home Assistant
-      image: home-assistant-mobile.png
+      image: HA-get-started-01.png
       content: |
         <ul>
-          <li class="info">Initial startup may take longer depending on your internet connection</li>
-          <li>Download the Home&nbsp;Assistant App on Android/IOS or visit <a href="http://homeassistant.local:8123" target="_blank">http://homeassistant.local:8123</a> in your phone's or computer's browser</li>
-          <li>Once your Home&nbsp;Assistant Yellow has finished setting up, the onboarding will guide you through setup on your phone or computer</li>
-          <li>For further information, refer to the instructions provided under <a href="https://www.home-assistant.io/getting-started/onboarding/" target="_blank">Onboarding</a></li>
+          <li class="info">Initial startup may take a while, depending on your internet connection.</li>
+          <li><b>Desktop</b>: Visit <a href="http://homeassistant.local:8123" target="_blank">http://homeassistant.local:8123</a> in your browser to access the web frontend.</li>
+          <li>The web frontend guides you through the initial onboarding.</li>
+          <li class="info">For further information, refer to the instructions provided under <a href="https://www.home-assistant.io/getting-started/onboarding/" target="_blank">https://www.home-assistant.io/getting-started/onboarding/</a>.</li></br>
+          <li><b>Installing the app</b></br>To access Home Assistant from your mobile device, use the QR code to locate the app in the app store.</li>
```

Comment:
Both work for initial onboarding, mobile or desktop. Can you move this point one up, and align it with Desktop (start with <b>Mobile</b>)

Reply:
Good point. Thank you, @agners .